src/libraries/Microsoft.Bcl.AsyncInterfaces/src/Microsoft.Bcl.AsyncInterfaces.csproj

```diff
@@ -1,7 +1,7 @@
 <Project Sdk="Microsoft.NET.Sdk">
 
   <PropertyGroup>
-    <TargetFrameworks>netstandard2.0;$(NetFrameworkMinimum);netstandard2.1</TargetFrameworks>
+    <TargetFrameworks>$(NetCoreAppMinimum);netstandard2.0;$(NetFrameworkMinimum);netstandard2.1</TargetFrameworks>
```

Comment:
This looks wrong. A package like Bcl.AsyncInterfaces shouldn't have a .NETCoreApp TFM. Its netstandard2.1 assembly only has typeforwards, there's no real implementation for .NETCoreApp as all the types are inbox. We shouldn't touch these TFMs.